Sources/App/Logger/LogLevel+Compare.swift

```diff
@@ -0,0 +1,33 @@
+import Logging
+
+extension LogLevel: Equatable {
+    public static func == (lhs: LogLevel, rhs: LogLevel) -> Bool {
+        return lhs.description == rhs.description
+    }
+
+    /// From more verbose to less verbose (i.e. from less important to more important)
+    private static let allLevels: [LogLevel] = [
+        .verbose, .debug, .info, .warning, .error, .fatal
+    ]
+
+    // Note: Didn't implement it as Comparable and `<` operator because `.custom` case doesn't follow semantics.
+    public func isAtLeast(minimumLevel: LogLevel) -> Bool {
+        // If minimumLevel is .custom, only keep the ones that are exactly using the same custom level name
+        if case .custom(let minValue) = minimumLevel, case .custom(let selfValue) = self {
```

Comment:
I'm starting to think we shouldn't have imported `Logging`, this feels overly complex.

Reply:
Yeah, that's the only thing that `Logging` provide that we might not want. But the `custom` case only affects this Compare logic, so reimplementing everything ourselves and then provide a bridge between our own implementation and Vapor _just_ to avoid that `.custom` case felt quite overkill to me 😉

Reply:
> so reimplementing everything ourselves and then provide a bridge between our own implementation and Vapor

Can you elaborate? I don't understand the part _reimplementing_ when we already had a protocol to abstract the logger, which is what `Logging` does so the only thing missing was having log levels which could be easily done by us as specified in the ticket, instead of having this train of levels we could have just 3 and avoid all of this complexity. 

Honestly I feel like we jumped into the implementation without checking the tickets and the expected AC and consider the best option based on that. 

Another thing we are losing is the abstraction from Vapor, `Bot` should be a module completely independent of Vapor and it really isn't anymore even if it's just for the logging part, however, as I mentioned feels like we brought an unnecessary layer of complexity without a need for it.